plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergOrcConnectorTest.java

```diff
@@ -21,40 +21,13 @@
 import java.nio.file.Path;
 
 import static com.google.common.io.Resources.getResource;
-import static io.trino.plugin.iceberg.IcebergFileFormat.ORC;
-import static io.trino.plugin.iceberg.IcebergTestUtils.checkOrcFileSorting;
 import static java.lang.String.format;
 import static java.nio.file.StandardCopyOption.REPLACE_EXISTING;
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 public class TestIcebergOrcConnectorTest
-        extends BaseIcebergConnectorTest
+        extends BaseIcebergOrcConnectorTest
```

Comment:
My PoV this test might be dropped. But we should move 
- public void testTinyintType()
- public void testSmallintType()
to `TestIcebergMinioConnectorTest` and maybe rename it to `TestIcebergORCMinioConnectorTest`

Reply:
yes to both. the tests need to survive and the class name needs to include "Orc" if its purpose is to cover ORC.